### PR TITLE
Dune compatibility

### PR DESCRIPTION
**Pull request by edwintorok**

### CODE REVIEW

**Review by lindig — APPROVED** [review]

File: Makefile [hunk 1]
```diff
@@ -1,6 +1,9 @@
 .PHONY: release
 release:
-	dune build --profile=release @install
+	dune build -p varstored-guard @install
```
Comment (lindig) [inline Makefile:3]: This does not have the `release` profile. Does this risk a performance problem?

Comment (edwintorok) [reply Makefile:3]: `-p` implies `--profile release`, Dune wouldn't let me use both of them:
```
dune build -p varstored-guard --profile=release
dune: Cannot use --profile and -p simultaneously
```

Comment (edwintorok) [reply Makefile:3]:
```
 -p PACKAGES, --for-release-of-packages=PACKAGES
           Shorthand for --root . --only-packages PACKAGE
           --ignore-promoted-rules --no-config --profile release.
```

Comment (psafont) [reply Makefile:3]: strange, it allows me it use both on dune 2.4.0

**Review by psafont — APPROVED** [review]

**Pull request merged**